server/eclipse-project/src/main/java/us/freeandfair/corla/csv/ColoradoBallotManifestParser.java

```diff
@@ -122,19 +137,17 @@ private BallotManifestInfo extractBMI(final CSVRecord the_line,
     BallotManifestInfo result = null;
     
     try {
-      // obtain the county from the ID or name in the line
-      final County c = CountyQueries.fromString(the_line.get(COUNTY_ID_COLUMN));
-      if (c != null) {
-        result = 
-            Persistence.get(new BallotManifestInfo(the_timestamp, 
-                                                   c.identifier(),
-                                                   the_line.get(SCANNER_ID_COLUMN),
-                                                   the_line.get(BATCH_NUMBER_COLUMN),
-                                                   Integer.parseInt(the_line.
-                                                                    get(NUM_BALLOTS_COLUMN)),
-                                                   the_line.get(BATCH_LOCATION_COLUMN)),
-                            BallotManifestInfo.class);
-      }
+      // TODO: should we check for mismatched county IDs between the
```

Comment:
It would, but for Dominion, not for us. It would mean that a county was uploading ballot manifests with the wrong county name in them.